plugins/main/public/components/common/welcome/agents-welcome.js

```diff
@@ -360,31 +446,37 @@ class AgentsWelcome extends Component {
           <EuiFlexItem>
             <EuiFlexGroup>
               <EuiFlexItem>
-                <h2 className="embPanel__title wz-headline-title">
-                  <EuiText size="xs"><h2>MITRE</h2></EuiText>
+                <h2 className='embPanel__title wz-headline-title'>
+                  <EuiText size='xs'>
+                    <h2>Events count evolution</h2>
+                  </EuiText>
                 </h2>
               </EuiFlexItem>
-              <EuiFlexItem grow={false} style={{ alignSelf: 'center' }}>
-                <EuiToolTip position="top" content="Open MITRE">
-                  <EuiButtonIcon
-                    iconType="popout"
-                    color="primary"
-                    onClick={() => {
-                      window.location.href = `#/overview?tab=mitre`;
-                      this.router.reload();
-                    }
-                    }
-                    aria-label="Open MITRE" />
-                </EuiToolTip>
-              </EuiFlexItem>
             </EuiFlexGroup>
+            <EuiSpacer size='s' />
+            <div
+              style={{
+                height: this.props.resultState !== 'loading' ? '350px' : 0,
+              }}
+            >
+              <WzReduxProvider>
+                <KibanaVis
+                  visID={'Wazuh-App-Agents-Welcome-Events-Evolution'}
+                  tab={'welcome'}
+                ></KibanaVis>
+              </WzReduxProvider>
+            </div>
+            <div
+              style={{
+                display:
+                  this.props.resultState === 'loading' ? 'block' : 'none',
+                alignSelf: 'center',
+                paddingTop: 100,
+              }}
+            >
+              <EuiLoadingChart size='xl' />
+            </div>
```

Comment:
**nitpick**: this seems there was a problem solving the conflict. The method should render a button related to MITRE ATT&CK. See the current logic in the target branch: https://github.com/wazuh/wazuh-kibana-app/blob/4.6.0-7.16/plugins/main/public/components/common/welcome/agents-welcome.js#L356-L392

Reply:
Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.